frontend/src/modules/products/components/post-product/PostProduct.tsx

```diff
@@ -1,17 +1,20 @@
 import './PostProduct.scss';
 import '@assets/styles/styles.scss';
-import {Form} from 'react-bootstrap';
-import React, {ReactElement} from 'react';
+import { Form, FormControl } from 'react-bootstrap';
```

Comment:
Replace `·Form,·FormControl·` with `Form,·FormControl`   
'FormControl' is defined but never used    
Replace `·ReactElement·` with `ReactElement`